### PR TITLE
Updated installation page of docs

### PR DESCRIPTION
**Pull request by matine**

### CODE REVIEW

File: src/templates/pages/installation.hbs [hunk 1]
```diff
@@ -16,15 +16,21 @@
...
                     <p>Git repo: <code>https://github.com/Pod-Point/pod-point-ui-toolkit</code></p>
-                    <p>Install via npm: <code>npm install git+ssh://git@github.com/Pod-Point/pod-point-ui-toolkit.git</code> <br>(when made public, will be able to npm install normally and save in package.json)</p>
-                    <p>Current version: (will be..) <code>Release 1.0</code></p>
+                    <p>Install via npm: <code>npm install @pod-point/ui-toolkit</code></p>
+                    <h4>See more docs for importing/copying files</h4>
+                    <ul class="remove-link-styles bold">
+                        <li><a href="scss.html#cssStructureImports">Importing styles</a></li>
+                        <li><a href="css-bg-img.html">Copying/relinking CSS background images</a></li>
+                        <li><a href="inline-svg-icons.html">Copying SVG icons/making inline</a></li>
+                        <li><a href="javascript.html">Importing javascript modules</a></li>
```
Comment (mikefrancis) [inline src/templates/pages/installation.hbs]: JavaScript

File: dist/installation.html [hunk 2]
```diff
@@ -81,15 +81,21 @@
                 <div class="container">
                     <h3>Installation</h3>
                     <p>The POD Point UI Toolkit is a node module package within the POD Point npm user</p>
-                    <p>Note: while the toolkit is still in beta, it is not published publicly, therefore needs to be installed via ssh with the following instructions</p>
                 </div>
             </header>
 
             <div class="content-block">
                 <div class="container">
+                    <h4>Importing the node package</h4>
```
Comment (mikefrancis) [inline dist/installation.html]: Node

File: dist/installation.html [hunk 3]
```diff
@@ -81,15 +81,21 @@
...
                     <p>Git repo: <code>https://github.com/Pod-Point/pod-point-ui-toolkit</code></p>
-                    <p>Install via npm: <code>npm install git+ssh://git@github.com/Pod-Point/pod-point-ui-toolkit.git</code> <br>(when made public, will be able to npm install normally and save in package.json)</p>
-                    <p>Current version: (will be..) <code>Release 1.0</code></p>
+                    <p>Install via npm: <code>npm install @pod-point/ui-toolkit</code></p>
```
Comment (mikefrancis) [inline dist/installation.html]: NPM

File: src/templates/pages/installation.hbs (hunk 1 above)
Comment (matine) [reply src/templates/pages/installation.hbs]: Dammit

**Pull request merged**